templates/zerver/integrations/github-actions.md

```diff
@@ -1,8 +1,60 @@
-It's easy to send Zulip messages from GitHub Actions! This is useful:
+# Zulip GitHub Actions integration
 
-  * If you want to report the success or failure of an Action to Zulip.
-  * If you want to send output from your actions to Zulip.
+Get Zulip notifications from GitHub Actions!
 
-Instructions are available [here](https://github.com/zulip/github-actions-zulip#readme).
+{start_tabs}
 
-See also the [GitHub integration](/integrations/doc/github).
+1.  {!create-a-generic-bot.md!}
+
+1.  To send Zulip notifications whenever your workflow runs, add the
+    `zulip/github-actions-zulip/send-message@v1` action to your GitHub
+    Actions workflow file, and set the input values as specified in the
+    sample templates below.
+
+    Use the details of the generic bot created above for the `api-key` and
+    `email` parameters.
+
+1.  Use the following template to send a message to a Zulip channel. This
+    example posts to the "Scheduled backups" topic in a Zulip channel named
+    "github-actions updates".
+
+      ```
+      {% raw %}- name: Send a channel message
+      uses: zulip/github-actions-zulip/send-message@v1
+      with:
+         api-key: ${{ secrets.ZULIP_API_KEY }}
```

Comment:
Does the user put the bot's API key inside the formatting here? Or is there a secrets file as part of the GitHub Actions set up? Or would the user put the API key string here in the template, like the email? If it's the latter, then I think it'd be clearer to just use a fake API key here.